src/test/java/lottery/domain/LotteryTicketsTest.java

```diff
@@ -14,15 +14,15 @@
     @DisplayName("생성 테스트")
     void validGenerationTest() {
         final List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6);
-        final LotteryTickets generatedTickets = LotteryTickets.create(1, FixedLotteryNumberGenerator.create(numbers));
-        final LotteryTickets expectedTickets = LotteryTickets.create(Arrays.asList(LotteryTicket.of(numbers)));
+        final LotteryTickets generatedTickets = LotteryTickets.create(PurchasePrice.from(1000), FixedLotteryNumberGenerator.create(numbers));
```

Comment:
```suggestion
        final LotteryTickets generatedTickets = LotteryTickets.create(PurchasePrice.from(1000), () -> numbers);
```
역시나 요런 방법도 있으니 참고해주세요ㅎㅎ